minio/parsers.py

```diff
@@ -143,6 +143,14 @@ def text(self):
         """
         return self.element.text
 
+    def is_dir(self):
+        """Returns True if the object is a dir
+        ie, if an object name has `/` suffixed.
+
+        """
+        text = self.get_child_text('Key')
+        return text.endswith("/")
+
```

Comment:
can you add a functional test for this as well

Reply:
yes .. ok

Reply:
Seems minio-py does not allow directory/empty puts , ie, we cannot do `mb play/test/testb` or `put obj/metaobj/` as per minio-py standards. 
AFAIK such puts can be done `mc` (`mc mb test/b/metab`) or `awscli`/`boto3`. So this test needs to be in mc/awscli. What say @nitisht ?

Reply:
Or adding unit test is fine ?